facebookads/objects.py

```diff
@@ -1440,6 +1450,11 @@ def _set_data(self, data):
         else:
             return super(AdImage, self)._set_data(data)
 
+    def export_data(self):
+        """Since AdImage is not updatable there's no need to collect
+        changed properties"""
+        return {}
```

Comment:
I'm not sure I agree with change. Is this required for this diff? If not, maybe submit it in a different PR to have a conversation around it. :)